Makefile

```diff
@@ -78,12 +78,12 @@ LIBS=be localestub translation bnetapi $(STDCPPLIBS)
 #	be recursive, so include all of the paths where libraries can
 #	be found.  Directories where source files are found are
 #	automatically included.
-LIBPATHS=
+LIBPATHS=/boot/system/develop/lib/ /boot/system/develop/lib/x86/
```

Comment:
Right, but likewise, it's confusing, ld will try to link to gcc4 libs if it doesn't find the gcc2 version, instead to give a proper error.